content/docs/concepts/flops/overview.md

```diff
@@ -0,0 +1,141 @@
+  To the best of our knowledge, FLOps is the first work that combines Flower with MLflow, allows hierarchical FL, and automatically converts ML code into FL-enabled containerized images.
+
+  Every third-party component and dependency has been carefully analyzed and compared with possible alternatives.
+  The following is a selection of tools and frameworks that power FLOps.
+
+  ### Flower
+  [Flower](https://flower.ai/) is the leading research-backed open-source FL framework on the market.
+  Due to Flower’s high flexibility and customization, FLOps was able to combine it with other tools and its own components to achieve novel features.
+  Flower implements and handles core FL components, but it does not handle many other necessary aspects such as deployment, orchestration, dependency management, containerization, or advanced tracking and observability tools seen in MLOps.
+
+  ### mlflow
+  [MLflow](https://mlflow.org/) is a mature open-source MLOps tool that powers the runtime observability and tracking features of FLOps.
+  MLflow augments and supports the entire ML lifecycle, from conception, to code and dependency archivation and unification, tracking and tagging ML models and artifacts, and re-redeployments. 
+
+  ### Apache's 'Data Suite'
+  FLOps works with real data on learner devices.
+  Several technologies provided by [Apache](https://www.apache.org/) are employed to efficiently receive, store, and retrieve such data.
+  - [Arrow](https://arrow.apache.org/docs/format/Columnar.html) columnar format for working with data in memory.
+  - [Parquet](https://parquet.apache.org/) columnar file format for working with data stored on disk.
+  - [Arrow Flight](https://arrow.apache.org/docs/format/Flight.html) a gRPC-based framework to transport Apache formatted data over the network.
+
+  ### Buildah
+  [Buildah](https://buildah.io/) helps FLOps to build containerized FL images inside running containers.
+  Therefore distributing the computational load of image building across devices.
+
+  ### Anaconda Suite
+  FLOps uses the [Anaconda Suite](https://docs.anaconda.com/) to resolve dependencies to provide equal ML training across devices.
+  To be more specific:
+   - [Miniconda](https://docs.anaconda.com/miniconda/) for lightweight (base) images
+   - [Mamba](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) for fast dependency resolving
+
+{{< /details >}}
+
+{{< details "**Convenient Installation & native CLI**" >}}
+  FLOps can be easily set up by cloning its [repository](https://github.com/oakestra/addon-FLOps) and running the docker-compose file on the same machine as the Oakestra Root Orchestrator.
+  See the [FLOps Manuals](/docs/manuals/flops) for concrete instructions.
+
+  The [Oakestra CLI](/docs/getting-started/deploy-app/with-the-cli/#the-oak-cli) supports a [set of commands](/docs/manuals/cli/features/flops-addon/#oak-addon-flops) to work with FLOps.
```

Comment:
Convert this to a `link-card`?

Reply:
This is sadly not possible AFAIK

Hugo does not like nested shortcodes:

![image](https://github.com/user-attachments/assets/d1aa8d53-1f8b-4e03-b385-4f061f026174)